README.md

```diff
@@ -198,6 +198,27 @@ in [Endpoints.scala](https://github.com/VirtusLab/ide-probe/tree/master/core/api
 
 Note that any communication with the probe is synchronous.
 
+## Screenshots
+
+Screenshots are saved under the path taken from `IDEPROBE_SCREENSHOTS_DIR` environment variable for which the default value is `/tmp/ide-probe/screenshots`. Inside this directory
```

Comment:
Okay... but as discussed previously... isn't `IDEPROBE_SCREENSHOTS_DIR` supposed to be an implementation detail, **not** to be exposed to end-users? 🤔 End-users should only care about `probe.paths.screenshots` config... whether there is some underlying env var passed b/w ide-probe and IntelliJ instance is irrelevant and might change in future ‼️